smooth-scroll/smooth-scroll-tests.ts

```diff
@@ -0,0 +1,57 @@
+//////////////////////////////////////////////////////////////////////////////
+// init Tests: https://github.com/cferdinandi/smooth-scroll#global-settings //
+//////////////////////////////////////////////////////////////////////////////
+namespace init_tests {
+    {
+        smoothScroll.init()
+        smoothScroll.init({})
+        smoothScroll.init({
+            selector: '[data-scroll]',
+            selectorHeader: '[data-scroll-header]',
+            speed: 500,
+            easing: 'easeInOutCubic',
+            offset: 0,
+            callback: function (anchor, toggle) {
+                let elem: Element;
+                elem = anchor;
+                toggle = toggle;
+            }
+        })
+    }
+}
+
+
+////////////////////////////////////////////////////////////////////////////////////////
+/// animateScroll Tests : https://github.com/cferdinandi/smooth-scroll#animatescroll ///
+////////////////////////////////////////////////////////////////////////////////////////
+namespace animate_scroll_tests {
+    // It's Example 1 from offical doc
+    {
+        let anchor = document.querySelector('#bazinga');
+        smoothScroll.animateScroll(anchor);
```

Comment:
@andy-ms I have a question. This line is checked by Travis CI. It shows `Argument of type 'Element | null' is not assignable to parameter of type 'number'.` but my local doesn't shoot the error. What happened?